package.json

```
@@ -40,8 +40,8 @@
     "typescript": "^4.3.4"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Maybe we can use `||` to allow versions starting at 16.8.0 and  17.0.0. This is the approach used in [react-query](https://github.com/tannerlinsley/react-query/blob/master/package.json#L73) and [swr](https://github.com/vercel/swr/blob/main/package.json#L112).
```suggestion
    "react": "^16.8.0 || ^17.0.0",
    "react-dom": "^16.8.0 || ^17.0.0"
```

---

@omarnyte This is great idea